app/assets/stylesheets/custom.scss

```diff
@@ -208,3 +209,16 @@ th {
 th {
   text-align: center;
 }
+
+li.active > a
+{
+  color: white;
+  text-decoration: none;
+  background-color: #093D5E !important;
+  outline: 0;
+}
+
+#lend-link, #signup, #loans, #logout, #profile {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Each selector in a comma sequence should be on its own line